src/work-allocation-2/containers/work-case-list/work-case-list.component.html

```diff
@@ -0,0 +1,98 @@
+<div *ngIf="pagination; else tableSummary" class="pagination-top" attr.aria-label="{{ casesTotal }} results have been found" role="status">
+  <span class="text-16" id="search-result-summary__text">Displaying
+      <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
+      -
+      <span class="govuk-!-font-weight-bold">{{ getLastResult() }}</span>
+      out of
+      <span class="govuk-!-font-weight-bold">{{ casesTotal || 0 }}</span>
+      cases.</span>
+</div>
+
+<ng-template #tableSummary>
+  <div class="pagination-top" attr.aria-label="{{ casesTotal }} results have been found" role="status">
+    <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{ casesTotal || 0 }} </span>cases.</span></div>
+</ng-template>
+
+<!-- Reset Sorting Button -->
+<div *ngIf="showResetSortButton" class="reset-sort-button"><button type="button" class="govuk-button hmcts-button--secondary" (click)="onResetSorting()">Reset sorting</button></div>
+
+<!-- Using the Angular Component Development Kit Table Module, and not Angular Material -->
+<table cdk-table [dataSource]="dataSource$" class="govuk-table" [multiTemplateDataRows]="true">
+
+  <!-- Column Headers and Cells -->
+<ng-container *ngIf="casesTotal">
+  <ng-container *ngFor="let field of fields | paginate: { itemsPerPage: pagination?.page_size, currentPage: pagination?.page_number, totalItems: casesTotal }" [cdkColumnDef]="field.name">
+    <th class="govuk-table__header" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
+      <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
+              (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
+    </th>
+    <td class="govuk-table__cell border-none" cdk-cell *cdkCellDef="let rowData">
+      <exui-work-field [workField]="rowData" [config]="field"></exui-work-field>
+    </td>
+  </ng-container>
+</ng-container>
+<!-- Below container sets up initial case list headers when not yet populated as session storage of page number causing issues with above container -->
+<ng-container *ngIf="!casesTotal">
+  <ng-container *ngFor="let field of fields" [cdkColumnDef]="field.name">
+    <th class="govuk-table__header" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
+      <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
+              (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
```

Comment:
_Nitpick:_
There's that extraneous space again.

Reply:
Removed.